templates/docs/base/forms.md

```diff
@@ -124,7 +124,10 @@ The `<input type="range">` allows a user to select from a specified range of val
 View example of the slider pattern
 </a></div>
 
-## Fieldset
+<h2>
+  Fieldset
+  <div class="p-status-label--information u-vertical-align--middle">Updated</div>
```

Comment:
A bit of chicken and egg problem here, but now that 4.16 is released, and this PR will be part of 4.17 all of those labels added here will not be relevant to new release anymore and could (should?) be removed.

I guess the only one relevant so far is the new utility itself.